src/mixins/legal-api-mixin.ts

```diff
@@ -94,15 +87,17 @@ export default class LegalApiMixin extends Vue {
   }
 
   /**
-   * Method to make a simple axios Get request.
-   * @param filingId Optional filing id if this resuming an existing draft.
+   * Method to make a simple axios Get request on the draft filings.
    */
-  private getFiling (filingId: number): Promise<any> {
+  private getDraftFiling (): Promise<any> {
     // Assign the url business identifier
-    let url = `${this.getBusinessIdentifier}/filings/${filingId}`
+    let url = `businesses/${this.getBusinessIdentifier}/tasks`
```

Comment:
Are we expecting a "tasks" endpoint for businesses? I hadn't thought so.

Reply:
What I mean is, isn't this supposed to be the same as getting a draft filing (businesses/<identifier>/filings/<filing id>)?

Reply:
Hmm, from my understanding when a client returns to the Incorporation flow, all we have is an identifier ( NRnumber for example and possibly the type ). So without a filingId, we needed to get the draft information and filingId ( from tasks as above ).

Also doing a GET on the `businesses/identifier/filings` will only return paid and or completed filings. (Not drafts, as far as i know )

We currently get all the draft information back from tasks as it's the only task filing that exists for that identifier, which is also retrieving it in one api request opposed to two.

I could be wrong here but was just working within what currently existed! Open to any suggestions 100% :)